cli/ingress-controller/main.go

```diff
@@ -196,9 +196,22 @@ func main() {
 		glog.Fatalf("Error creating Kong Rest client: %v", err)
 	}
 
-	root, err := kongClient.Root(context.Background())
-	if err != nil {
-		glog.Fatalf("%v", err)
+	// fix: https://github.com/Kong/kubernetes-ingress-controller/issues/761
+	var root map[string]interface{}
+	retry := 0
+	for {
+		root, err = kongClient.Root(context.Background())
+		if err != nil {
+			if retry > 5 {
+				glog.Fatalf("%v", err)
+			} else {
+				retry++
+				time.Sleep(time.Second * 5)
```

Comment:
We should use exponential backoff instead of a static sleep here.

You can refer how it is done in the following file for reference:
https://github.com/Kong/kubernetes-ingress-controller/blob/master/internal/ingress/election/election.go#L40-L49